### ISSUE TITLE
File already exists error

### ISSUE THREAD

**Issue by Madis0** [RozeFound/mmc-export#23]
This should either prompt or just recreate it lol, not throw an error
```
Traceback (most recent call last):
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\Scripts\mmc-export.exe\__main__.py", line 7, in <module>
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\lib\site-packages\mmc_export\main.py", line 75, in main
    loop.run_until_complete(run())
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\lib\site-packages\mmc_export\main.py", line 64, in run
    writer.write()
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\lib\site-packages\mmc_export\Formats\modrinth.py", line 72, in write
    Path(archive_name).rename(archive_name.replace("zip", "mrpack"))
  File "C:\Users\Madis\AppData\Local\Programs\Python\Python310\lib\pathlib.py", line 1232, in rename
    self._accessor.rename(self, target)
FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\Users\\Madis\\Desktop\\MR_Fabulously Optimized.zip' -> 'C:\\Users\\Madis\\Desktop\\MR_Fabulously Optimized.mrpack'
````

**Comment by RozeFound** [comment]
Interesting, never had such a problem

**Comment by RozeFound** [comment]
Maybe it's OS dependent...

**Comment by RozeFound** [comment]
Yep, it definietly is: https://docs.python.org/3/library/pathlib.html#pathlib.Path.rename
> On Unix, if target exists and is  > a file, it will be replaced silently if the user has permission. On Windows, if target exists, FileExistsError will be raised. target can be either a string or another path object:

**Comment by RozeFound** [comment]
Fixed with dd8353aa16d459576a1c7b03667c83d77d802ccb
